niyontwali/updated-portfolio-nextjs
# Next.js Authentication

## Description of the PR

In this PR, I added the next authentication that will be used in protection of different routes on my portfolio page revamping

## Testing the functionalities in the PR

1.  Clone the repo
2.  Run `npm i` to install all package
3.  Check in the root directory a `.env sample` in order to include it in your `.env` file
4.  Run `npm run dev` to start your development server'
5.  Go to the site and try to login (I used google login provider). Therefore, use a google account i.e gmail
